src/com/alevel/hw6/Car.java

```diff
@@ -0,0 +1,91 @@
+package com.alevel.hw6;
+
+class Car {
+    private static final int MAX_FUEL = 100;
+
+    final private Manufacturer manufacturer;
```

Comment:
The usual order: private final field, but I'm not sure if a final is needed here